src/types.js

```diff
@@ -22,6 +22,18 @@ export type Mode = String | {
 
 export type AlignOpts = "top" | "center" | "bottom";
 
+export type ObjectInspectorItemContentsValue = {
+    actor: string,
+    class: string,
+    displayClass: string,
+    extensible: boolean,
+    frozen: boolean,
+    ownPropertyLength: number,
+    preview: Object,
+    sealed: boolean,
+    type: string
+};
```

Comment:
I think i'd prefer having `ObjectInspector.js` or `object-inspector` export the type.